doc/explanation/networks.md

```diff
@@ -1,7 +1,107 @@
+{ref}`network-bridge`
+: A network bridge creates a connection between different networks or network segments.
+
+  In LXD context, the `bridge` network type creates an L2 bridge that connects the instances that use it and makes it possible to pass traffic between them.
+  The bridge can also provide local DHCP and DNS.
+
+  This is the default network type.
+
+{ref}`network-ovn`
+: {abbr}`OVN (Open Virtual Network)` is a software-defined networking system that supports virtual network abstraction.
+  You can use it to build your own private cloud.
+  See https://www.ovn.org/ for more information.
+
+  In LXD context, the `ovn` network type creates a logical network.
+  To set it up, you must install and configure the OVN tools.
+  In addition, you must create an uplink network that provides the network connection for OVN.
+  As the uplink network, you should use one of the external network types.
+
+### External networks
+
+External networks use network interfaces that already exist.
+Therefore, LXD has limited possibility to control them.
+
+The main purpose for using external networks is to provide an uplink network through a parent interface.
+
+LXD supports the following network types:
+
+{ref}`network-macvlan`
+: Macvlan is a virtual {abbr}`LAN (Local Area Network)` that can assign several IP addresses to the same network interface, basically splitting up the network interface into several sub-interfaces with their own IP addresses.
+  The IP addresses are assigned based on randomly generated MAC addresses.
+
+  In LXD context, the `macvlan` network type provides a preset configuration to use when connecting instances to a parent Macvlan interface.
+
+{ref}`network-sriov`
+: {abbr}`SR-IOV (Single root I/O virtualization)` is a hardware standard that allows a single network interface to appear as several virtual network interfaces in a virtualized environment.
+
+  In LXD context, the `sriov` network type provides a preset configuration to use when connecting instances to a parent SR-IOV interface.
+
+{ref}`network-physical`
+: The `physical` network type connects to an existing physical network, which can be a network card or a bridge, and serves as an uplink network for OVN.
+
+  It provides a preset configuration to use when connecting OVN networks to a parent interface.
+
+## Recommendations
+
+Which network type to choose depends on your specific use case.
+In general, if you can use a fully controlled network, you should do so because it provides more functionality than using a network device.
+In addition, networks are easy to configure, and you can reuse the same network for several instances without repeating the configuration.
+
+As a general recommendation:
+
+- If you are running LXD on a single system or in a public cloud, use a {ref}`network-bridge`, possibly in connection with the [Ubuntu Fan](https://www.youtube.com/watch?v=5cwd0vZJ5bw).
+- If you are running LXD on your own private cloud, use an {ref}`network-ovn`.
+
```

Comment:
Maybe add a point here:

> - Use the instance NIC `network` property to connect to a managed network rather than `parent` so that the NIC can inherit the settings from the network and `nictype` doesn't have to be specified.